code/game/machinery/exp_cloner.dm

```diff
@@ -69,6 +71,24 @@
 		H.set_cloned_appearance()
 
 		H.set_suicide(FALSE)
+
+		if(!isnull(H.mind) && prob(defect_chance)) // If we don't have mind, don't set any objectives to it, or it will break the cloner with an infinite loop of exceptions.
+			if(defect_chance < 100)
+				defect_chance = defect_chance + 10
```

Comment:
`defect_chance += 10`

Reply:
Doesn't matter

Reply:
Do it

Reply:
no?

Reply:
I code a lot in lua, lua doesn't have that thing, so I don't use it what so ever.

Reply:
Yes I'm sure you use loads of Lua in your Roblox scripts but we use `+=` here